### PR TITLE
WIP: Add short move shards guide

### PR DESCRIPTION
**Pull request by jukomericki**
When horizontal scaling feature is released for the cloud, we would also like to give our customers some information on how they can easily move their collection shards from their old nodes to their new nodes

### CODE REVIEW

File: qdrant-landing/content/documentation/cloud/cluster-scaling.md [hunk 1]
```diff
@@ -35,3 +35,46 @@ With the growing amount of data you can add nodes to your cluster and move shard
 
 We will be glad to consult you on an optimal strategy for scaling.
 [Let us know](mailto:cloud@qdrant.io) your needs and decide together on a proper solution. We plan to introduce an auto-scaling functionality. Since it is one of most desired features, it has a high priority on our Cloud roadmap.
+
+### Guide: Moving shards to a new node after adding nodes
+
+After scaling a cluster horizontally, the user can move shards from their old node to the new one to balance the memory and storage utilization of the nodes.
```
Comment (fabriziobonavita) [inline qdrant-landing/content/documentation/cloud/cluster-scaling.md:41]:
```suggestion
After scaling a cluster horizontally, you can move shards from your original nodes to the new ones to balance memory and storage utilization. 
```

File: qdrant-landing/content/documentation/cloud/cluster-scaling.md [hunk 2]
```diff
@@ -35,3 +35,46 @@ With the growing amount of data you can add nodes to your cluster and move shard
 
 We will be glad to consult you on an optimal strategy for scaling.
 [Let us know](mailto:cloud@qdrant.io) your needs and decide together on a proper solution. We plan to introduce an auto-scaling functionality. Since it is one of most desired features, it has a high priority on our Cloud roadmap.
+
+### Guide: Moving shards to a new node after adding nodes
+
+After scaling a cluster horizontally, the user can move shards from their old node to the new one to balance the memory and storage utilization of the nodes. 
+
+NOTE: this guide is an example of moving shards from node 0, if the user wants to move shard from another
```
Comment (fabriziobonavita) [inline qdrant-landing/content/documentation/cloud/cluster-scaling.md:43]:
```suggestion
NOTE: this guide is an example of moving shards from node 0, if you want to move shards from another 
```

File: qdrant-landing/content/documentation/cloud/cluster-scaling.md [hunk 3]
```diff
@@ -35,3 +35,46 @@ With the growing amount of data you can add nodes to your cluster and move shard
 
 We will be glad to consult you on an optimal strategy for scaling.
 [Let us know](mailto:cloud@qdrant.io) your needs and decide together on a proper solution. We plan to introduce an auto-scaling functionality. Since it is one of most desired features, it has a high priority on our Cloud roadmap.
+
+### Guide: Moving shards to a new node after adding nodes
+
+After scaling a cluster horizontally, the user can move shards from their old node to the new one to balance the memory and storage utilization of the nodes. 
+
+NOTE: this guide is an example of moving shards from node 0, if the user wants to move shard from another 
+node they would need to change their cluster URL to `https://node-{source_node}-xyz-example.eu-central.aws.cloud.qdrant.io`
```
Comment (fabriziobonavita) [inline qdrant-landing/content/documentation/cloud/cluster-scaling.md:44]:
```suggestion
node you would need to change your cluster URL to `https://node-{source_node}-xyz-example.eu-central.aws.cloud.qdrant.io`
```

File: qdrant-landing/content/documentation/cloud/cluster-scaling.md [hunk 4]
```diff
@@ -35,3 +35,46 @@ With the growing amount of data you can add nodes to your cluster and move shard
...
+
+Firstly, after adding nodes confirm that new peers are added by running the following:
```
Comment (fabriziobonavita) [inline qdrant-landing/content/documentation/cloud/cluster-scaling.md:46]:
```suggestion
Firstly, after adding nodes confirm that new peers are present by running:
```

File: qdrant-landing/content/documentation/cloud/cluster-scaling.md [hunk 5]
```diff
@@ -35,3 +35,46 @@ With the growing amount of data you can add nodes to your cluster and move shard
...
+
+```bash
+curl \
+  -X GET 'https://node-0-xyz-example.eu-central.aws.cloud.qdrant.io:6333/cluster' \
+  --header 'api-key: <paste-your-api-key-here>'
+```
+
+Note the `peer_ids` of the peers as they are needed for the last step of moving shards
+
+Afterwards the user needs to check the target collection cluster info. This gives insights of how many shards are on the node and its size. Note the `local_shards` from the return result and note a `shard_id` of the shard you want to move to another node.
```
Comment (fabriziobonavita) [inline qdrant-landing/content/documentation/cloud/cluster-scaling.md:56]:
```suggestion
Afterwards you need to check the target collection cluster info. This gives insights of how many shards are on the node and its size. Note the `local_shards` from the return result and note a `shard_id` of the shard you want to move to another node.
```

File: qdrant-landing/content/documentation/cloud/cluster-scaling.md [hunk 6]
```diff
@@ -35,3 +35,46 @@ With the growing amount of data you can add nodes to your cluster and move shard
...
+  --header 'api-key: <paste-your-api-key-here>' -d '{
+    "move_shard": {
+    "shard_id": SHARD_ID,
+    "from_peer_id": SOURCE_PEER_ID,
+    "to_peer_id": TARGET_PEER_ID
+  }
+}' -H "Content-Type: application/json"
+```
+
+This inititates a background shard move operation which doesn't affect the database performance. After the shard is moved the user
```
Comment (fabriziobonavita) [inline qdrant-landing/content/documentation/cloud/cluster-scaling.md:78]: It can actually affect performance as it'll increase cpu on the target node

Comment (fabriziobonavita) [inline qdrant-landing/content/documentation/cloud/cluster-scaling.md:78]:
```suggestion
This initiates a background shard move operation. After the shard is moved you
```

File: qdrant-landing/content/documentation/cloud/cluster-scaling.md [hunk 7]
```diff
@@ -35,3 +35,46 @@ With the growing amount of data you can add nodes to your cluster and move shard
...
+can confirm it by once again running the second step command (querying collection cluster info), and the user will notice that the shard has moved to another peer.
```
Comment (fabriziobonavita) [inline qdrant-landing/content/documentation/cloud/cluster-scaling.md:79]:
```suggestion
can confirm it by once again running the command from step two (querying collection cluster info), and you will notice that the shard has moved to another peer.
```

**Review by fabriziobonavita — CHANGES_REQUESTED** [review]
Great! Left some suggestions

File: qdrant-landing/content/documentation/cloud/cluster-scaling.md [hunk 8]
```diff
@@ -35,3 +35,45 @@ With the growing amount of data you can add nodes to your cluster and move shard
 
 We will be glad to consult you on an optimal strategy for scaling.
 [Let us know](mailto:cloud@qdrant.io) your needs and decide together on a proper solution. We plan to introduce an auto-scaling functionality. Since it is one of most desired features, it has a high priority on our Cloud roadmap.
+
+### Guide: Moving shards to a new node after adding nodes
+
+After scaling a cluster horizontally, you can move shards from your original nodes to the new ones to balance memory and storage utilization.
+
+NOTE: NOTE: this guide is an example of moving shards from node 0, if you want to move shards from another
```
Comment (bashofmann) [inline qdrant-landing/content/documentation/cloud/cluster-scaling.md:43]:
```suggestion
NOTE: This guide is an example of moving shards from node 0, if you want to move shards from another 
```

File: qdrant-landing/content/documentation/cloud/cluster-scaling.md [hunk 9]
```diff
@@ -35,3 +35,45 @@ With the growing amount of data you can add nodes to your cluster and move shard
...
 [Let us know](mailto:cloud@qdrant.io) your needs and decide together on a proper solution. We plan to introduce an auto-scaling functionality. Since it is one of most desired features, it has a high priority on our Cloud roadmap.
+
+### Guide: Moving shards to a new node after adding nodes
+
+After scaling a cluster horizontally, you can move shards from your original nodes to the new ones to balance memory and storage utilization.
+
+NOTE: NOTE: this guide is an example of moving shards from node 0, if you want to move shards from another 
+node you would need to change your cluster URL to `https://node-{source_node}-xyz-example.eu-central.aws.cloud.qdrant.io`
+
+Firstly, after adding nodes confirm that new peers are present by running:
```
Comment (bashofmann) [inline qdrant-landing/content/documentation/cloud/cluster-scaling.md:46]:
```suggestion
First, after adding nodes confirm that new peers are present by running:
```

File: qdrant-landing/content/documentation/cloud/cluster-scaling.md [hunk 10]
```diff
@@ -35,3 +35,45 @@ With the growing amount of data you can add nodes to your cluster and move shard
...
+
+```bash
+curl \
+  -X GET 'https://node-0-xyz-example.eu-central.aws.cloud.qdrant.io:6333/cluster' \
+  --header 'api-key: <paste-your-api-key-here>'
+```
+
+Note the `peer_ids` of the peers as they are needed for the last step of moving shards
```
Comment (bashofmann) [inline qdrant-landing/content/documentation/cloud/cluster-scaling.md:54]:
```suggestion
Note the `peer_ids` of the peers as they are needed for the last step of moving shards.
```

File: qdrant-landing/content/documentation/cloud/cluster-scaling.md [hunk 11]
```diff
@@ -35,3 +35,45 @@ With the growing amount of data you can add nodes to your cluster and move shard
...
+
+Afterwards you need to check the target collection cluster info. This gives insights of how many shards are on the node and its size. Note the `local_shards` from the return result and note a `shard_id` of the shard you want to move to another node.
```
Comment (bashofmann) [inline qdrant-landing/content/documentation/cloud/cluster-scaling.md:56]:
```suggestion
Next, you need to check the target collection cluster info. This gives insights on how many shards are on a node and its size. Note the `local_shards` from the return result and note a `shard_id` of the shard you want to move to another node.
```

File: qdrant-landing/content/documentation/cloud/cluster-scaling.md [hunk 12]
```diff
@@ -35,3 +35,45 @@ With the growing amount of data you can add nodes to your cluster and move shard
...
+  --header 'api-key: <paste-your-api-key-here>' -d '{
+    "move_shard": {
+    "shard_id": SHARD_ID,
+    "from_peer_id": SOURCE_PEER_ID,
+    "to_peer_id": TARGET_PEER_ID
+  }
+}' -H "Content-Type: application/json"
+```
+
+This initiates a background shard move operation. After the shard is moved you can confirm it by once again running the command from step two (querying collection cluster info), and you will notice that the shard has moved to another peer.
```
Comment (bashofmann) [inline qdrant-landing/content/documentation/cloud/cluster-scaling.md:78]:
```suggestion
This initiates a background operation to move a shard. After the shard is moved, you can confirm the operation's success by once again running the command from step two (querying collection cluster info), and you will notice that the shard has moved to another peer.
```

**Comment by bashofmann** [comment]
I proposed a couple changes to fix grammar and style.

**Comment by bashofmann** [comment]
Not needed. This will be done automatically by cluster-manager in cloud.

**Pull request closed without merging**